tests/Acceptance/Service/RequestTest.php

```diff
@@ -0,0 +1,140 @@
+use GuzzleHttp\Psr7\Response;
+use GuzzleHttp\Psr7\Uri;
+use PHPUnit\Framework\TestCase;
+
+class RequestTest extends TestCase
+{
+    /**
+     * @test
+     * @dataProvider \Answear\LuigisBoxBundle\Tests\DataProvider\RequestDataProvider::forContentUpdate()
+     */
+    public function contentUpdateRequestPassed(
+        string $httpMethod,
+        ContentUpdateCollection $collection,
+        string $expectedContent
+    ): void {
+        $response = $this->getRequestService($httpMethod, $expectedContent)->contentUpdate(
+            $collection
+        );
+
+        $this->assertSame(200, $response->getStatusCode());
+    }
+
+    /**
+     * @test
+     * @dataProvider \Answear\LuigisBoxBundle\Tests\DataProvider\RequestDataProvider::forPartialContentUpdate()
+     */
+    public function partialContentUpdateRequestPassed(
+        string $httpMethod,
+        ContentUpdateCollection $collection,
+        string $expectedContent
+    ): void {
+        $response = $this->getRequestService($httpMethod, $expectedContent)->partialContentUpdate(
+            $collection
+        );
+
+        $this->assertSame(200, $response->getStatusCode());
+    }
+
+    /**
+     * @test
+     * @dataProvider \Answear\LuigisBoxBundle\Tests\DataProvider\RequestDataProvider::forContentRemoval()
+     */
+    public function contentRemovalRequestPassed(
+        string $httpMethod,
+        ContentRemovalCollection $collection,
+        string $expectedContent
+    ): void {
+        $response = $this->getRequestService($httpMethod, $expectedContent)->contentRemoval(
+            $collection
+        );
+
+        $this->assertSame(200, $response->getStatusCode());
+    }
+
+    /**
+     * @test
+     * @dataProvider \Answear\LuigisBoxBundle\Tests\DataProvider\RequestDataProvider::forChangeAvailability()
+     */
+    public function changeAvailabilityRequestPassed(string $httpMethod, $collection, string $expectedContent): void
+    {
+        $response = $this->getRequestService($httpMethod, $expectedContent)->changeAvailability(
+            $collection
+        );
+
+        $this->assertSame(200, $response->getStatusCode());
+    }
+
+    private function getRequestService(string $httpMethod, string $expectedContent): Request
+    {
+        $endpoint = '/v1/content';
+
+        $configProvider = new ConfigProvider('host', '', '', 5.0, 5.0);
+
+        $expectedRequest = new \GuzzleHttp\Psr7\Request(
+            $httpMethod,
+            new Uri('host' . $endpoint),
+            [
+                'Content-Type' => ['application/json; charset=utf-8'],
+                'date' => [''],
+                'Authorization' => [''],
+            ],
+            $expectedContent
+        );
+        $serializer = new LuigisBoxSerializer();
+
+        $client = $this->createMock(Client::class);
+        $client->expects($this->once())
+            ->method('request')
+            ->with(
+                $this->callback(
```

Comment:
check on callback, because headers data and Authorization is change on every request, and response objects will be not the same.